c2corg_api/scripts/migration/documents/waypoints/summit.py

```diff
@@ -70,6 +72,13 @@ def get_document_locale(self, document_in, version):
             summary=summary
         )
 
+    def get_climbing_outdoor_types(self, document_in):
+        outdoor_types = []
+        if document_in.summit_type == 4:
+            outdoor_types.append('multi')
+        outdoor_types = outdoor_types if outdoor_types else None
+        return outdoor_types
```

Comment:
I simply copied the structure of this function : https://github.com/c2corg/v6_api/blob/master/c2corg_api/scripts/migration/documents/waypoints/sites.py#L112-L141
I didn't try understanding why each line has been coded like this.
You can change the code as you want.